src/Components/AllCandidates/AllCandidates.jsx

```diff
@@ -221,16 +221,18 @@ function AllCandidates({ history, program }) {
           <Header>
             <h1 style={{ color: "black" }}>All Candidates</h1>
             <div className="button-container">
-              <Button
-                variant="contained"
-                color="primary"
-                target="_blank"
-                value="CreateForm"
-                style={{ width: "250px", maxWidth: "250px" }}
-                onClick={initiateForm}
-              >
-                Create form
-              </Button>
+              {/*
+               // <Button
+               //   variant="contained"
+               //   color="primary"
+               //   target="_blank"
+               //   value="CreateForm"
+               //   style={{ width: "250px", maxWidth: "250px" }}
+               //   onClick={initiateForm}
+               // >
+               //   Create form
+               // </Button>
+               */}
```

Comment:
The create form "button" should be a part of the top right dropdown now right? Or do we still want to have a button in the future when that feature is done?